tailwind.config.js

```diff
@@ -3,7 +3,10 @@ module.exports = {
     // removeDeprecatedGapUtilities: true,
     // purgeLayersByDefault: true,
   },
-  purge: [],
+  purge: [
+    './app/views/**/*.haml',
+    './app/javascript/**/*',
+  ],
```

Comment:
@ErikSchierboom @kntsoriano @SleeplessByte Are these the only places that will have CSS class names? I think so.

Reply:
If you ensure you don't write html on the controller, model, decorator, etc. side, and dont use the regular asset pipeline, then yes.